firefox-ios/Storage/Rust/RustRemoteTabs.swift

```diff
@@ -30,6 +35,8 @@ public class RustRemoteTabs {
     private func open() -> NSError? {
         store = TabsStore(path: databasePath)
         isOpen = true
+        commandQueue = RemoteTabsCommandQueue()
```

Comment:
Nit: Shall we add a matching `commandQueue = nil` to `close()`?

Reply:
https://github.com/mozilla-mobile/firefox-ios/pull/22799/commits/7d44018db816a8b4494ca1d7b1f12e8113690ce9